source/agent/conference/conference.js

```diff
@@ -575,7 +587,10 @@ var Conference = function (rpcClient, selfRpcId) {
       var participant = participants[participantId];
       var left_user = participant.getInfo();
       delete participants[participantId];
-      room_config.notifying.participantActivities && sendMsg('room', 'all', 'participant', {action: 'leave', data: left_user.id});
+      if (room_config.notifying.participantActivities) {
+        sendMsg('room', 'all', 'participant', {action: 'leave', data: left_user.id});
+        !participants[participantId].cascading && sendMsgTo('cascading', 'participant', {type: "addParticipant", rpcId: selfRpcId, data: participantId});
```

Comment:
`type: "addParticipant"` in function `removeParticipant`?
Also follow current style using '' instead of "" for string.